build-uboot-rpi64.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2024 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Script to generate Mender integration binaries for Raspberry Pi boards
+#
+# Files that will be packaged:
+#
+#     - u-boot.bin
+#     - fw_printenv
+#     - fw_env.config
+#
+# NOTE! This script is not necessarily well tested and the main purpose
+# is to provide an reference on how the current integration binaries where
+# generated.
+
+set -e
+
+function usage() {
+    echo "./$(basename $0) <defconfig> <board name>"
+}
+
+if [ -z "$1" ] || [ -z "$2" ]; then
+    usage
+    exit 1
+fi
+
+# Availabile defconfigs:
+#
+#    - rpi_3_defconfig
+#    - rpi_4_defconfig
+#    - rpi_arm64_defconfig
+#
+rpi_defconfig=$1
+rpi_board=$2
+
+echo "$rpi_board: starting"
+
+# ARM 64bit build
+export CROSS_COMPILE=aarch64-linux-gnu-
+export ARCH=aarch64
+
+# Test if the toolchain is actually installed
+aarch64-linux-gnu-gcc --version
```

Comment:
I suggest reusing the variable to avoid typos:
```suggestion
${CROSS_COMPILE}gcc --version
```

Reply:
Done.